composer.json

```diff
@@ -5,7 +5,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "behat/behat": "^3.4",
-        "psr/container": "^1.0",
+        "psr/container": "^1.0 || ^2.0",
```

Comment:
`composer.lock` needs updating too :thinking:

Reply:
@Ocramius Sorry for missing that. Latest commit addressing `composer.lock` update